packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -520,4 +521,32 @@ export class MetriportMedicalApi {
     const resp = await this.api.post(url, docRef);
     return resp.data;
   }
+
+  /**
+   * Verifies the signature of a webhook request.
```

Comment:
signature verif. Not positive that this is the best place for it since it is the only function in this file that isn't making an http request